examples/nlp/md/multiple_choice_task_with_transfer_learning.md

```diff
@@ -0,0 +1,895 @@
+│ (<span style="color: #0087ff; text-decoration-color: #0087ff">Concatenate</span>)       │                   │         │ deberta_v3_classifi… │
+│                     │                   │         │ deberta_v3_classifi… │
+│                     │                   │         │ deberta_v3_classifi… │
+├─────────────────────┼───────────────────┼─────────┼──────────────────────┤
+│ softmax (<span style="color: #0087ff; text-decoration-color: #0087ff">Softmax</span>)   │ (<span style="color: #00d7ff; text-decoration-color: #00d7ff">None</span>, <span style="color: #00af00; text-decoration-color: #00af00">4</span>)         │       <span style="color: #00af00; text-decoration-color: #00af00">0</span> │ concatenate[<span style="color: #00af00; text-decoration-color: #00af00">0</span>][<span style="color: #00af00; text-decoration-color: #00af00">0</span>]    │
+└─────────────────────┴───────────────────┴─────────┴──────────────────────┘
+</pre>
+
+
+
+
+<pre style="white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace"><span style="font-weight: bold"> Total params: </span><span style="color: #00af00; text-decoration-color: #00af00">70,830,337</span> (270.20 MB)
+</pre>
+
+
+
+
+<pre style="white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace"><span style="font-weight: bold"> Trainable params: </span><span style="color: #00af00; text-decoration-color: #00af00">70,830,337</span> (270.20 MB)
+</pre>
+
+
+
+
+<pre style="white-space:pre;overflow-x:auto;line-height:normal;font-family:Menlo,'DejaVu Sans Mono',consolas,'Courier New',monospace"><span style="font-weight: bold"> Non-trainable params: </span><span style="color: #00af00; text-decoration-color: #00af00">0</span> (0.00 B)
+</pre>
+
+
+
+Finally, let's check the model structure visually if everything is in place.
+
+
+```python
+keras.utils.plot_model(model, show_shapes=True)
+```
+
+<div class="k-default-codeblock">
+```
+You must install pydot (`pip install pydot`) for `plot_model` to work.
+
+```
+</div>
+---
+## Training
+
+
+```python
+# Start training the model
+history = model.fit(
+    train_ds,
+    epochs=CFG.epochs,
+    validation_data=valid_ds,
+    callbacks=callbacks,
+    steps_per_epoch=int(len(train_df) / CFG.batch_size),
+    verbose=1,
+)
+```
+
+<div class="k-default-codeblock">
+```
+/opt/conda/envs/keras-jax/lib/python3.10/site-packages/keras/src/backend/jax/numpy.py:132: UserWarning: Explicitly requested dtype int64 requested in arange is not available, and will be truncated to dtype int32. To enable more dtypes, set the jax_enable_x64 configuration option or the JAX_ENABLE_X64 shell environment variable. See https://github.com/google/jax#current-gotchas for more.
```

Comment:
What is the cause of this warning? The input layer in the code is explicitly int32.

Reply:
[This line](https://github.com/keras-team/keras-nlp/blob/11bece833200548c181810980ff50546da4d748a/keras_nlp/models/deberta_v3/disentangled_self_attention.py#L240) causes the problem by explicitly use `dtype="int64"`.
@mattdangerw Any idea on why we did this?

Reply:
Probably just because this was written for tensorflow first, and on tensorflow int64 plays nicer with accelerators.

It's a bug though for the Keras 3 world. I think we can just update that to `dtype="int"` and get int32 for jax and int64 for tf with this line https://github.com/keras-team/keras/blob/91a50cfff9a6010b75fff8649088cc9cd7c514c8/keras/backend/common/variables.py#L366

Reply:
Is to possible to stick to `int32` everywhere? It's more consistent.

Reply:
Re: the current example, if we fix KerasNLP then we can just remove the warning by hand here.

Reply:
Cool. We will also need a small fix in keras for it to work. I am creating the PR.

According to the discussion with Matt, TF works better with int64? That is why we try to use the implicit "int" here.

Reply:
I had a problem with pyplot and graphviz, too. Need to regenerate the file.